doc/api/v8.md

```diff
@@ -22,6 +22,15 @@ Returns an object with the following properties:
 * `total_available_size` {number}
 * `used_heap_size` {number}
 * `heap_size_limit` {number}
+* `malloced_memory` {number}
+* `peak_malloced_memory` {number}
+* `does_zap_garbage` {number}
+
+does_zap_garbage is a 0/1 boolean, which signifies whether the zap_code_space
```

Comment:
Can you put does_zap_garbage and zap_code_space between backticks and put two dashes before the latter?  People will have a hard time guessing it's a command line switch otherwise.